tools/generate_report.rb

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env ruby
+require File.expand_path('../config/environment', __dir__)
+
+include ActionDispatch::Routing::UrlFor
+include Rails.application.routes.url_helpers
```

Comment:
Normally, I'd be concerned with including this like this directly in self but this is a one off script you run alone.